### ISSUE TITLE
Export setting

### ISSUE THREAD

**Issue by Bobcadence** [vtoubiana/TrackMeNot#96]
It would be great to have export/import feature for settings.
In deed, my setup is resetting every firefox update.

**Comment by dhowe** [comment]
good idea -- a simple json import/export of settings would be a nice enhancement
